roles/confluent.ssl/tasks/self_signed_certs.yml

```diff
@@ -32,7 +32,7 @@
       -storetype pkcs12 \
       -keyalg RSA -keysize 2048 \
       -alias {{inventory_hostname}} \
-      -dname "CN={{inventory_hostname}},OU=TEST,O=CONFLUENT,L=PaloAlto,S=Ca,C=US" \
+      -dname "CN={{service_name}},OU=TEST,O=CONFLUENT,L=PaloAlto,S=Ca,C=US" \
```

Comment:
@domenicbove have we tested this change with RBAC?  I suspect there maybe some changes required in our testing scenarios.

Reply:
Good thinking, I just ran rbac-mtls-rhel to make sure this wouldn't impact, it succeeded. the kafka rbac set principal code takes the DNAME out of each kafka's keystore and uses that as a principal in the super user list, so changing the dname *should* be fine. But absolutely worth testing